databricks_cli/unity_catalog/ext_loc_cli.py

```diff
@@ -0,0 +1,165 @@
+    click.echo(mc_pretty_format(locs_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Get an external location.')
+@click.option('--name', required=True,
+              help='Name of the external location to get.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def get_location_cli(api_client, name):
+    """
+    Get an external location.
+
+    Calls the 'getExternalLocation' RPC endpoint of the Unity Catalog service.
+    Returns an ExternalLocation object.
+
+    """
+    loc_json = UnityCatalogApi(api_client).get_external_location(name)
+    click.echo(mc_pretty_format(loc_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Update an external location.')
+@click.option('--name', required=True,
+              help='Name of the external location to update.')
+@click.option('--force', '-f', is_flag=True, default=False,
+              help='Force update even if location has dependent tables/mounts')
+@click.option('--json-file', default=None, type=click.Path(),
+              help='File containing JSON request to PATCH.')
+@click.option('--json', default=None, type=JsonClickType(),
+              help=JsonClickType.help('/api/2.0/external-locations'))
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def update_location_cli(api_client, name, force, json_file, json):
+    """
+    Update an external location.
+
+    Calls the 'updateExternalLocation' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    json_cli_base(json_file, json,
+                  lambda json: UnityCatalogApi(api_client).update_external_location(name, force,
+                                                                                    json),
+                  encode_utf8=True)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Delete an external location.')
+@click.option('--name', required=True,
+              help='Name of the external location to delete.')
+@click.option('--force', '-f', is_flag=True, default=False,
+              help='Force deletion even if location has dependent tables/mounts')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def delete_location_cli(api_client, name, force):
+    """
+    Delete an external location.
+
+    Calls the 'deleteExternalLocation' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    UnityCatalogApi(api_client).delete_external_location(name, force)
+
+
+def register_ext_loc_commands(cmd_group):
+    cmd_group.add_command(create_location_cli, name='create-external-location')
+    cmd_group.add_command(list_locations_cli, name='list-external-locations')
+    cmd_group.add_command(get_location_cli, name='get-external-location')
+    cmd_group.add_command(update_location_cli, name='update-external-location')
+    cmd_group.add_command(delete_location_cli, name='delete-external-location')
```

Comment:
please make command naming consistent with the rest of the CLI (see comments for other command groups)

Reply:
Addressed in #471.